specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2021-10-01/costmanagement.json

```diff
@@ -1115,7 +1115,7 @@
           "ReservedInstances"
         ],
         "operationId": "GenerateReservationDetailsReport_ByBillingAccountId",
-        "description": "Generates the reservations details report for provided date range asynchronously based on enrollment id.",
+        "description": "Generates the reservations details report for provided date range asynchronously based on enrollment id. The Reservation usage details can be viewed by only certain enterprise roles by default(Enterprise Administrator, EA Purchaser and Partner). For more details on the roles see, https://docs.microsoft.com/en-us/azure/cost-management-billing/manage/understand-ea-roles#usage-and-costs-access-by-role",
```

Comment:
You can remove the examples of EA Administrator, Partner, Purchaser if they are covered in the link.